homeassistant/components/dsmr/config_flow.py

```diff
@@ -290,9 +308,14 @@ async def async_validate_dsmr(
         data = input_data
 
         try:
-            info = await _validate_dsmr_connection(self.hass, data)
+            try:
+                is_rfxtrx = False
```

Comment:
Please only wrap the line that can raise in the try... except block. Move this up out of the try except.

Reply:
Sure, updated.